src/parser.ts

```diff
@@ -164,82 +179,171 @@ class Parser {
         return null;
     }
 
-    public getPropsInfo(propsObj: ts.Symbol): Props {
+    public getPropsInfo(propsObj: ts.Symbol, defaultProps: StringIndexedObject<string> = {}): Props {
         const propsType = this.checker.getTypeOfSymbolAtLocation(propsObj, propsObj.valueDeclaration);
         const propertiesOfProps = propsType.getProperties();
 
         const result: Props = {};
 
         propertiesOfProps.forEach(prop => {
-            const propName = prop.getName();            
+            const propName = prop.getName();
 
             // Find type of prop by looking in context of the props object itself.
             const propType = this.checker.getTypeOfSymbolAtLocation(prop, propsObj.valueDeclaration);
-            
+
             const propTypeString = this.checker.typeToString(propType);
 
             const isOptional = (prop.getFlags() & ts.SymbolFlags.Optional) !== 0;
 
 
             const jsDocComment = this.findDocComment(prop);
 
+            let defaultValue = null;
+
+            if (defaultProps[propName] !== undefined) {
+                defaultValue = { value: defaultProps[propName] };
```

Comment:
I know it's an old code but I'm wondering why is the `defaultValue` was made an object with `value` key and not the value itself?